source/develop/release-management/features/engine/engine-snmp3.html.md

```diff
@@ -0,0 +1,119 @@
+---
+title: SNMPv3
+category: feature
+authors: rnori
+feature_name: Add support to send SNMPv3 traps from notifier
+feature_modules: engine
+feature_status: In Development
+---
+# engine-snmp
+
+## oVirt Engine SNMPv3 Traps
+
+See also [Features/configuration-event-subscribers](Features/configuration-event-subscribers).
+
+### Summary
+
+This feature extends events notifier capabilities and enables it to generate SNMPv3 traps out of system events to integrate oVirt with generic monitoring systems.
+
+### Owner
+
+*   Name: [Ravi Nori](User:rnori)
+*   Email: <rnori@redhat.com>
+
+### Current status
+
+*   Target Release: 4.1
+*   Status: Post
+*   Last updated: 
+
+### Configuration
+
+Add a new configuration file /etc/ovirt-engine/notifier/notifier.conf.d/20-snmp.conf to configure the SNMP.
```

Comment:
Please change file name 99-snmp.conf, AFAIR all user customizations should begin with number higher than 60